libs/s25main/GameObject.cpp

```diff
@@ -28,7 +28,7 @@
 unsigned GameObject::objIdCounter_ = 0;
 unsigned GameObject::objCounter_ = 0;
 
-GameWorldGame* GameObject::gwg = nullptr;
+GameWorld* GameObject::world = nullptr;
```

Comment:
we can't really get rid of this static world efficiently, do we?

Reply:
Every gameobject needs access to it somehow. With a lot of refactoring we could pass that into the HandleEvent function which then passes it down to where it is needed. Or pass it through all ctors and store in GameObject. Likely easier.